src/main/java/org/onedatashare/server/controller/UserController.java

```diff
@@ -1,99 +1,99 @@
-                                      userAction.getOrganization());
-      case "validate":
-        return userService.validate(userAction.getEmail(), userAction.getCode());
-      case "history":
-        return userService.saveHistory(userAction.getUri(), cookie);
-      case "verifyEmail":
-        return userService.verifyEmail(userAction.getEmail());
-      case "sendVerificationCode":
-        return userService.sendVerificationCode(userAction.getEmail(), TIMEOUT_IN_MINUTES);
-      case "getUser":
-        return userService.getUserFromCookie(userAction.getEmail(), cookie);
-      case "getUsers":
-        return userService.getAllUsers(userAction, headers.getFirst(ODSConstants.COOKIE));
-      case "getAdministrators":
-        return userService.getAdministrators(userAction, cookie);
-      case "verifyCode":
-        return userService.verifyCode(userAction.getEmail(), userAction.getCode());
-      case "setPassword":
-        return userService.resetPassword(userAction.getEmail(), userAction.getPassword(), userAction.getConfirmPassword(),
-                                          userAction.getCode());
-      case "resetPassword":
-        return userService.resetPasswordWithOld(cookie, userAction.getPassword(), userAction.getNewPassword(),
-                                                  userAction.getConfirmPassword());
-      case "deleteCredential":
-        return userService.deleteCredential(cookie, userAction.getUuid());
-      case "deleteHistory":
-        return userService.deleteHistory(cookie, userAction.getUri());
-      case "isAdmin":
-        return userService.isAdmin(cookie);
-      case "resendVerificationCode":
-        return userService.resendVerificationCode(userAction.getEmail());
-      default:
-        return null;
-    }
-  }
-  @PutMapping
-  public Object putAction(@RequestHeader HttpHeaders headers, @RequestBody UserAction userAction){
-    switch(userAction.getAction()) {
-      case "updateAdminRights":
-        return userService.updateAdminRights(userAction.getEmail(), userAction.isAdmin());
-      default:
-        return null;
-    }
-  }
-
-  @ExceptionHandler(NotFound.class)
-  public ResponseEntity<NotFound> handle(NotFound notfound) {
-    return new ResponseEntity<>(notfound, notfound.status);
-  }
-
-  @GetMapping
-  public Object getHistory(@RequestHeader HttpHeaders headers) {
-    return userService.getHistory(headers.getFirst(ODSConstants.COOKIE));
-  }
-
-  @ExceptionHandler(InvalidField.class)
-  public ResponseEntity<InvalidField> handle(InvalidField invf){
-    ODSLoggerService.logError(invf.getMessage());
-    return new ResponseEntity<>(invf, invf.status);
-  }
-
-  @ExceptionHandler(ForbiddenAction.class)
-  public ResponseEntity<ForbiddenAction> handle(ForbiddenAction fa){
-    ODSLoggerService.logError(fa.getMessage());
-    return new ResponseEntity<>(fa, fa.status);
-  }
-}
+package org.onedatashare.server.controller;
```

Comment:
Whats the change in this file?

Reply:
I don't see any changes on the diffs, I don't think there's any